plugin.video.vtm.go/resources/lib/kodilogging.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""Log handler for Kodi"""
+
+from __future__ import absolute_import, division, unicode_literals
+
+import logging
+
+import xbmc
+import xbmcaddon
+
+ADDON = xbmcaddon.Addon()
+
+
+class KodiLogHandler(logging.StreamHandler):
+    """ A log handler for Kodi """
+
+    def __init__(self):
+        logging.StreamHandler.__init__(self)
+        formatter = logging.Formatter("[{}] [%(name)s] %(message)s".format(ADDON.getAddonInfo("id")))
+        self.setFormatter(formatter)
+
+    def emit(self, record):
+        """ Emit a log message """
+        levels = {
+            logging.CRITICAL: xbmc.LOGFATAL,
+            logging.ERROR: xbmc.LOGERROR,
+            logging.WARNING: xbmc.LOGWARNING,
+            logging.INFO: xbmc.LOGNOTICE,
```

Comment:
Doesn't exist in matrix, please use loginfo

Reply:
Does LOGINFO also exist in Leia? I haven't published this addon to Matrix yet.

Reply:
I can't change this yet for Kodi 18, since `LOGINFO` [doesn't seem to be logged](https://forum.kodi.tv/showthread.php?tid=353818&pid=2944096#pid2944096) to file in Leia. That's why I do the mapping from DEBUG to LOGINFO during troubleshooting.

I will do this for Kodi Matrix though.

Reply:
yeah it looks like it was introduced only for matrix, please keep it in mind and disregard for now. I pointed this out as I've see some if/else clauses along the code already focusing on matrix.

Reply:
The code should be almost compatible with Matrix, it's the plan to have one codebase for both Matrix and Leia (hence the if/else cases). I will modify it to use LOGINFO for Matrix in a future release. Thanks for the heads-up anyway!